src/utils/helpers.ts

```diff
@@ -52,3 +52,7 @@ export function getMessageExchange(message: Message): string {
     // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
     return message.properties.headers?.[ORIGIN_EXCHANGE_HEADER] ?? message.fields.exchange;
 }
+
+export function appInTestingMode(): boolean {
+    return process.env.NODE_ENV === 'test';
```

Comment:
@Leichtwind I'd move it into a CqrsModule configurations settings (ex: connection string), rather than linking with env variable

Reply:
I thought about that as well. But let's release it as is to unblock people who have this issue and I'll do it with the another PR